src/backend/distributed/sql/udfs/create_distributed_function/latest.sql

```diff
@@ -1,7 +1,12 @@
-CREATE OR REPLACE FUNCTION create_distributed_function(function_name regprocedure)
+CREATE OR REPLACE FUNCTION create_distributed_function(function_name regprocedure,
+						       colocate_with text DEFAULT 'default',
+						       distribution_argument_name text DEFAULT '',
```

Comment:
Seems fine. I would have expected that the arguments have the same order as create_distributed_table in the happy case, but could be convinced otherwise.